packages/lesswrong/lib/sql/CreateIndexQuery.ts

```diff
@@ -19,14 +19,15 @@ import { StringType } from "./Type";
 class CreateIndexQuery<T extends DbObject> extends Query<T> {
   private isUnique: boolean;
 
-  constructor(table: Table<T>, index: TableIndex<T>, ifNotExists = true) {
+  constructor(table: Table<T>, index: TableIndex<T>, ifNotExists = true, allowConcurrent=true) {
```

Comment:
Rather than having multiple optional booleans, I think it'd be cleaner to have an object with named options